lib/assets/terms_template.json.erb

```diff
@@ -1,106 +1 @@
 LAW, WE DISCLAIM ALL WARRANTIES, STATUTORY, EXPRESS OR IMPLIED, INCLUDING IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, AND NON-INFRINGEMENT UNDER NO CIRCUMSTANCES WILL WE BE LIABLE FOR ANY SPECIAL, INDIRECT, INCIDENTAL, CONSEQUENTIAL, OR EXEMPLARY DAMAGES (INCLUDING WITHOUT LIMITATION, LOSS OF PROFITS, DATA, OR USE OR COST OF COVER) RESULTING FROM YOUR USE OR THE INABILITY TO USE THE SITE OR ITS CONTENTS, EVEN IF WE WERE AWARE OF THE POSSIBILITY OF SUCH DAMAGES. IN NO EVENT WILL WE HAVE ANY LIABILITY TO YOU FOR ALL CLAIMS OR DAMAGES FOR ANY REASON IN EXCESS OF ONE HUNDRED DOLLARS ($100 USD). CERTAIN JURISDICTIONS DO NOT ALLOW EXCLUSIONS OR LIMITATIONS ON IMPLIED WARRANTIES OR LIMITATIONS ON LIABILITY, AND THEREFORE SOME OR ALL OF THE DISCLAIMERS, EXCLUSIONS, OR LIMITATIONS MAY NOT APPLY TO YOU, AND YOU MAY HAVE ADDITIONAL RIGHTS.</p>\\\"\"},{\"id\":1575546404475,\"type\":\"text\",\"content\":\"\\\"<h2>4. Copyright Protection</h2>\\\"\"},{\"id\":1575546425422,\"type\":\"text\",\"content\":\"\\\"<p>If You are a copyright owner or an agent thereof, and believe that any content on the Site infringes Your copyrights, You may send a notice in writing to World Resources Institute Tel: + 1 (202) 729-7600 <% if contact_email.present? %>E-mail: <a href=\\\\\\\"mailto:<%= contact_email %>\\\\\\\" target=\\\\\\\"_self\\\\\\\"><%= contact_email %></a> <% end %>with the following information:</p><ol><li>Your signature as a person authorized to act on behalf of the owner of an exclusive right that is allegedly infringed;</li><li>Identification of the exact location on the Site of copyrighted work claimed to have been infringed, or, if multiple copyrighted works on the Site are covered by a single notification, a representative list of such works as located on the Site;</li><li>The address, telephone number, and e-mail address of the complaining party;</li><li>A statement that the complaining party has a good faith belief that use of the material in the manner complained of is not authorized by the copyright owner, its agent, or the law; and</li><li>A statement that the information in the notification is accurate, and under penalty of perjury, that the complaining party is authorized to act on behalf of the owner of an exclusive right that is allegedly infringed.</li></ol>\\\"\"},{\"id\":1575546511777,\"type\":\"text\",\"content\":\"\\\"<h2>5. Additional Terms</h2>\\\"\"},{\"id\":1575546528969,\"type\":\"text\",\"content\":\"\\\"<ol><li>If we do not promptly enforce the Terms, We have not given up any other rights or the right to enforce them or other Terms in the future.</li><li>If any particular provision of these Terms is not enforceable, that doesn't affect the enforceability of any other provision of the Terms.</li><li>These Terms are between You and Us, and no one else can enforce them.</li><li>These Terms are the entire statement of the agreement between You and Us regarding the Site and its contents.</li></ol>\\\"\"}]"
```

Comment:
@jesusgonzalezrivera I'm not sure about this part, can you check it?

```erb
<% if contact_email.present? %>E-mail: <a href=\\\\\\\"mailto:<%= contact_email %>\\\\\\\" target=\\\\\\\"_self\\\\\\\"><%= contact_email %></a> <% end %>
```

Note that right before the `<% end %>` part, there is a space. Will that work?